packages/bun-uws/src/HttpRouter.h

```diff
@@ -35,43 +35,7 @@ namespace uWS {
 
 template <class USERDATA>
 struct HttpRouter {
-    /* These are public for now */
-    std::vector<std::string> upperCasedMethods = {
```

Comment:
this reminded me that bun currently doesnt support exotic method strings like `walawala`, instead defaulting them to get. but this diff is just uws updating so it ok.